extension/scripts/main.js

```diff
@@ -69,30 +102,58 @@ function loadWidget () {
 	close.dataset.trackingAction = 'close';
 	close.addEventListener('click', removeSelf, false);
 
-
 	holder.setAttribute('id', 'cpp-widget-holder');
 	document.body.appendChild(holder);
 
-	return {
+	update();
+
+	widget = {
 		close: removeSelf,
+		update
 	}
 }
 
-chrome.runtime.sendMessage({
-	method: 'isEnabled',
-	host: location.host
-}, response => {
-	if (response.enabled) loadWidget();
-});
+function beginTests () {
```

Comment:
Yeah, I want all the tests to be started from the devtools, to avoid tests being triggered accidentially on the client.